packages/visual-editor/src/editor/README.md

```
@@ -231,19 +232,20 @@ type MyComponentProps = {
+    hideNumberFieldRadioLabel: msg("fields.options.all", "All"),
+    showNumberFieldRadioLabel: msg("fields.options.limit", "Limit"),
     defaultCustomValue: 3,
-  }),
+  },
 };
 ```
 
 ## YextField
 
-`YextField` provides a unified utility for creating typed field configurations in a [Puck](https://github.com/measuredco/puck) and Yext Visual Editor integration context. It abstracts over common field types and includes special handling for the internal `basicSelector` field type, [OptionalNumberField](##OptionalNumberField), [YextEntityFieldSelector](##YextEntityFieldSelector), and [TranslatableStringField](##TranslatableStringField).
+`YextField` provides a unified utility for creating typed field configurations in a [Puck](https://github.com/measuredco/puck) and Yext Visual Editor integration context. It abstracts over common field types and includes special handling for the internal `basicSelector` field type, [YextEntityFieldSelector](##YextEntityFieldSelector), and [TranslatableStringField](##TranslatableStringField).
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)